kivy/uix/widget.py

```diff
@@ -1319,19 +1320,19 @@ def set_disabled(self, value):
             else:
                 self.dec_disabled()
 
```

Comment:
Maybe you should `return True` here when new value for `self._disabled_value` is set? Because of `set` method in `AliasProperty`